.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,47 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+  pull_request:
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.19'
```

```suggestion
          go-version: '1.20'
```

The `go.mod` file says it is 1.20 for this project.

---

Or you can specify the same value as the coverage one.